packages/utilities/src/TextUtils.ts

```diff
@@ -150,19 +150,22 @@ export class TextUtils {
      * @param {boolean}  includeHeader - should the table include a header of the field names of the objects
      * @param includeBorders -  should the table have borders between the cells?
      * @param hardWrap - hard wrap the text within the width of the table cells (defaults to false)
+     * @param headers - specify which headers in which order to display. if omitted, loops through the rows
+     *        and adds object properties as headers in their enumeration order
      * @returns {string} the rendered table
      */
     public static getTable(objects: any[], primaryHighlightColor: string,
                            maxColumnWidth?: number, includeHeader: boolean = true, includeBorders: boolean = false,
-                           hardWrap: boolean = false): string {
+                           hardWrap: boolean = false, headers: string[] = []): string {
```

Comment:
it'd be nice to have an interface for this function so that people don't have to `getTable(objects, "white", null, true, false, false, ["header1","header2"])`, and instead call it like 
```
getTable({
  objects,
  primaryHighlightColor: "white",
  includeHeader: true,
  headers: ["header1", "header2"]
})
```